.deploy/naiserator.yaml

```diff
@@ -0,0 +1,71 @@
+apiVersion: "nais.io/v1alpha1"
+kind: "Application"
+metadata:
+  name: familie-dokument
+  namespace: teamfamilie
+  labels:
+    team: teamfamilie
+
+spec:
+  image: {{ image }}
+  team: teamfamilie
+  port: 8082
+  envFrom:
+    - secret: {{ secret }}
+    - configmap: loginservice-idporten
+  liveness:
+    path: /internal/status/isAlive
+    initialDelay: 30
+    timeout: 5
+    periodSeconds: 10     # How often (in seconds) to perform the probe. Default to 10 seconds
+    failureThreshold: 10 # when a Pod starts and the probe fails,
+  readiness:
+    path: /internal/status/isAlive
+    initialDelay: 30
+    timeout: 5
+    periodSeconds: 10
+    failureThreshold: 10
+  prometheus:
+    enabled: true
+    path: /internal/prometheus
+  vault:
+    enabled: true
+  replicas:
+    min: 2
+    max: 4
+    cpuThresholdPercentage: 80
+  resources:
+    limits:
+      memory: "1024Mi"
```

Comment:
Dette blir 256MB minne for applikasjonen. Fordelingen er 75% til pod og 25% til app. 
Det tror jeg er litt for lite hvis vi skal konvertere bilder til pdf.

Forslag; `memory: 2Gi`  og i Dockerfile kan vi justere til 50/50 deling mellom pod og app: `ENV JAVA_OPTS="-XX:MaxRAMPercentage=50"`

Reply:
Thank you for the information. I do not know the memory cosumption of  image convertor. I'll follow your advice!

Reply:
I made some memory changes here  here: https://github.com/navikt/familie-dokument/pull/39/files
Check 
- Dockerfile
- Memory changes in `app-*prod.yml`